be/src/olap/olap_cond.cpp

```diff
@@ -601,25 +601,24 @@ bool Conditions::delete_conditions_eval(const RowCursor& row) const {
 }
 
 bool Conditions::delta_pruning_filter(
-        const std::vector<std::pair<WrapperField*, WrapperField*>>& column_statistics) const {
+        const std::vector<std::pair<WrapperField*, WrapperField*>>& zone_maps) const {
```

Comment:
use KeyRange to confirm to the statement of this function

Reply:
OK